backend/src/dex/data.ts

```diff
@@ -71,11 +82,35 @@ export async function getUniqueAssetsData() {
     where tl.asset_code is not null;
   `;
 
-  const payments = await fetchCachedData("dex-uniqueAssets", query);
+  const payments = await fetchCachedData(
+    REDIS_DEX_KEYS.DEX_UNIQUE_ASSETS,
+    query,
+  );
 
   return payments[0]?.data;
 }
 
+export async function getActiveAccountsData() {
+  const query = `
+    SELECT COUNT(DISTINCT(op_source_account)) as data
+    FROM ${bigQueryEndpointBase}.enriched_history_operations 
+    where type in (1, 2, 6, 13)
+    AND closed_at >= TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -24 HOUR);
+  `;
+
+  const activeAccounts = await fetchCachedData(
+    REDIS_DEX_KEYS.DEX_ACTIVE_ACCOUNTS,
+    query,
+  );
+
+  return activeAccounts[0]?.data;
+}
+
+//TODO: Currently unused due to a pending update to how we fetch asset prices. Once they're available we should refactor and make use of the function below again.
+const cache24hKey = "dex-volume-sum-24h";
```

Comment:
I moved this stuff to the bottom of the file under this TODO comment, confused me we're not using this (yet)